scripts/deny

```diff
@@ -2,8 +2,12 @@
 
 set -euo pipefail
 
+# use a standalone rust version to build and run deny to decouple from the code build environment
+RUST_VERSION="1.92.0"
```

Comment:
Is it good to fix this version?

Reply:
It's the last stable version now. I think it's fine.